scripts/caf_route_to_local_k8s.sh

```diff
@@ -166,7 +166,7 @@ function f_add_routing() {
 
   echo "Press ctrl-c to exit"
   # We don't use 'infinity' as Macs don't support it
-  sleep 99999999
+  sleep ${CAF_ROUTE_TO_LOCAL_K8S_TIMEOUT:-99999999}
```

Comment:
Not sure you actually want a timeout this long!

Reply:
I think that's not really a timeout but an "infinite wait" unless control-c is pressed

Reply:
@bdellegrazie  @joan-s-molas is right here since the script is meant to run infinity and terminated via user interruption. I tried some ways to sent the SIGINT but to no avail so instead I made the sleep configurable via an environment variable.